index-pl.html

```diff
@@ -121,31 +121,31 @@ <h2>Say it, Don't Spray It</h2>
     <p>
     </p>
 
-    <h2>Why Masks Protect Us Twice</h2>
+    <h2>Dlaczego maski chronią nas podwójnie</h2>
 
     <p>
-      Let’s imagine that a contagious person wears a 50 percent effective mask.
-      By '50 percent effective', I mean that wearing this mask cuts in half the chance that they'll infect a nearby susceptible person.
+      Wyobraźmy sobie, że osoba zarażona nosi maskę o skuteczności 50%.
+      Przez "maskę o skuteczności 50%" mam na myśli to, że noszenie takiej maski obniża o połowę szansę, że osoba zarażona zarazi kogoś innego.
     </p>
 
   </div>
 
-  <anim-with-caption :mask1="true" :mask2="false" :eout="0.5" contagiousperson="Contagious Person" susceptibleperson="Susceptible Person"><span class="orange">Transmission Route 2:</span> When the contagious person wears a 50% effective mask, disease transmission drops by 50%.</anim-with-caption>
+  <anim-with-caption :mask1="true" :mask2="false" :eout="0.5" contagiousperson="Osoba zarażona" susceptibleperson="Osoba zdrowa"><span class="orange">2 sposób transmisji:</span> Kiedy osoba zarażona nosi maskę o skuteczności 50%, szanse zakażenia spadają o 50%.</anim-with-caption>
 
   <div class="container">
 
-    <p>But what if the susceptible person wears the mask instead?</p>
+    <p>Lecz co się dzieje jeśli zamist osoby chorej maskę nosi osoba zdrowa?</p>
 
     <p>
-      In general, the effectiveness of a mask depends on whether you’re <b>inhaling</b> or <b>exhaling</b> through it.
-      For now, let’s keep things simple and assume that this mask is equally effective in either direction.
+      Ogóle rzecz ujmując, skuteczność maski jest inna podczas <b>wdechu</b> oraz <b>wydechu</b>.
+      Na razie jednak nie wdawajmy się w szczegóły i przyjmijmy, że maska jest równie skuteczna w obu przypadkach.
```

Comment:
nie wchodźmy w szczegóły ?

Reply:
Muszę? To są synonimy.

Reply:
"Ogóle"? "Ogólnie"?

Reply:
*skuteczność maski różni się podczas wdechu i wydechu
?